package.json

```diff
@@ -121,6 +121,7 @@
     "rollup-plugin-license": "2.6.0",
     "rollup-plugin-size-snapshot": "0.12.0",
     "rollup-plugin-typescript2": "0.31.1",
+    "storybook-addon-designs": "6.2.0",
```

Comment:
This package connects Figma to Storybook. https://help.figma.com/hc/en-us/articles/360045003494-Storybook-and-Figma